controllers/managedocs_controller.go

```diff
@@ -95,6 +147,15 @@ func (r *ManagedOCSReconciler) reconcilePhases(req ctrl.Request) error {
 	}
 	r.managedOCS.Status.ReconcileStrategy = reconcileStrategy
 
+	configSecret := &corev1.Secret{}
+	err := r.Get(r.ctx, types.NamespacedName{Namespace: req.Namespace, Name: r.ConfigSecretName}, configSecret)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			return err
+		} else {
+			configSecret = nil
```

Comment:
Let's add a print to log here (info) and return to prevent reconciliation if the secret does not exists